### ISSUE TITLE
Should <link> elements be removed after reading?

### ISSUE THREAD

**Issue by jwadhams** [petecoop/laravel-inky#4]
Given a very simple template:
```html
<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Strict//EN" "http://www.w3.org/TR/xhtml1/DTD/xhtml1-strict.dtd">
<html xmlns="http://www.w3.org/1999/xhtml">
<head>
  <meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
  <meta name="viewport" content="width=device-width"/>
  <link rel="stylesheet" href="foundation-inky.css">
</head>
<body>
  <container>
    <row>
      <columns>The time is now, [[date("c")]]</columns>
    </row>
  </container>
</body>
</html>
```

The CSS file is correctly found in `resources/css` and correctly inlined.  This library is so awesome, I'm delighted to be using it.

But the HTML still contains a `<link>` element, and because the file it refers to is in a non-shared directory, it always 404s.

Is there something the library could do, e.g., near line 32 of `src/InkyCompilerEngine.php` to delete the `<link>` elements in the returned HTML?

(I'm testing locally by just returning the HTML like a regular view, and testing with MailTrap.io, both of which actually do try to follow the link.  I wouldn't be surprised if many "real" clients don't, but it still seems worthy of clean up.)

**Comment by petecoop** [comment]
Ah thanks for reporting that, I wasn't aware. Yeah it does need cleaning up!

**Comment by petecoop** [comment]
Hey I've now fixed this and published it as 0.3.0

**Comment by jwadhams** [comment]
This works great, thank you!
